src/xcode/ENA/ENA/Source/Models/Exposure/MockENManager.swift

```diff
@@ -71,16 +81,22 @@ final class MockENManager: NSObject {
 	// MARK: - Obtaining Exposure Keys
 
 	func getDiagnosisKeys(completionHandler: @escaping ENGetDiagnosisKeysHandler) {
+		// swiftlint:disable:next force_unwrapping
```

Comment:
Uh, could you please guard & unwrap and remove then the force unwrapping?
I know, it's annoying but we try to avoid force unwraps as much as we can.

Reply:
Sure, you are right. Not much benefit in using swiftlint if you then disable its warnings 🙃